EpikCord/client/commands.py

```diff
@@ -0,0 +1,57 @@
+from typing import List, Optional
+
+from ..flags import Permissions
+from ..locales import Localization
+from ..utils import AsyncFunction, ApplicationCommandType, localization_list_to_dict
+
+
+class BaseClientCommand:
+    def __init__(
+        self,
+        name: str,
+        # description: str,
+        callback: AsyncFunction,
+        *,
+        guild_ids: List[int] = [],
+        name_localizations: List[Localization] = [],
+        # description_localizations: List[Localization] = [],
```

Comment:
why all those comments everywhere

Reply:
Trying to figure out what works for context menu commands.